pkg/kubelet/kubelet_pods.go

```diff
@@ -207,16 +207,17 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 				return nil, cleanupAction, fmt.Errorf("unable to provision SubPath `%s`: %v", mount.SubPath, err)
 			}
 
-			fileinfo, err := os.Lstat(hostPath)
+			volumePath, err := mounter.GetAbsoluteHostPath(hostPath)
```

Comment:
This needs to be carefully integrated with https://github.com/kubernetes/kubernetes/pull/61480 to ensure that we convert the path back to the host's path @gnufied

Reply:
This smells like another opportunity for resolution races. Is any part of this path under the user's control?

Reply:
This path is the base volume's path, which for most volumes is "/var/lib/kubelet/..." and under our control.  hostpath volumes, however, are not.